crates/nu-cli/src/commands/math/avg.rs

```diff
@@ -108,6 +125,28 @@ pub fn average(values: &[Value], name: &Tag) -> Result<Value, ShellError> {
                 &name.span,
             )
         })
+    } else if is_duration {
+        to_duration(&sum(
+            UntaggedValue::int(0).into_untagged_value(),
```

Comment:
Does sum not work over durations directly? If not, I wonder if we could update it to. That way, you wouldn't need to convert in and out of Int.